skeleton-typescript-webpack/src/main.ts

```diff
@@ -0,0 +1,14 @@
+﻿import {Aurelia} from 'aurelia-framework';
+import {bootstrap} from 'aurelia-bootstrapper-webpack';
```

Comment:
I haven't run the code yet, but does this work for you? In my local version using Webpack, I had to include this using `require` instead of an import for some reason. It could be an issue with my local setup though. I'll pull down your changes and run them.